collectiondb.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+$db = connectToDB("pokemon");
+$pokemon = collectData($db);
```

Comment:
I don't really like how the $pokemon in this file is brought into the main file, it would be better instead to have the $pokemon = collectData() line in the main file instead

Reply:
Moved into the main file :)